package.json

```diff
@@ -8,8 +8,11 @@
     "build": "lerna run clean && lerna run build",
     "test": "lerna run --parallel test",
     "test:watch": "lerna run --parallel test:watch --no-prefix",
-    "lint:check": "eslint .",
-    "lint:fix": "eslint --fix .",
+    "lint:staged": "lerna run lint-staged && :",
```

Comment:
What is this `&& :` doing?

Reply:
supposed to pass the long concatenation of staged file paths into the bash no-op command `:`